schema.development.kf

```diff
@@ -97,8 +80,47 @@ table delegates {
     foreign_key (inserter_id) references inserters(id) on_delete cascade
 }
 
+table access_grants {
+    id uuid primary,
+    ag_owner text notnull,
+    ag_grantee text notnull,
+    data_id text notnull,
+    locked_until int,
```

Comment:
We should make this `notnull`. Maybe we also want other field to not be null, but I know I want this one present for sure.

Reply:
Good point. Done.